boards/arm/frdm_kw41z/doc/frdm_kw41z.rst

```diff
@@ -207,3 +207,29 @@ program your Zephyr application to flash. It will leave you at a gdb prompt.
 
 .. _Segger J-Link OpenSDA V2.1 Firmware:
    https://www.segger.com/downloads/jlink/OpenSDA_V2_1.bin
+
+BLE Support
+***********
+
+Native BLE support in Zephyr on the NXP KW41Z SOC requires linking
+with the NXP provided BLE Link Layer Controller library. The library will need to
```

Comment:
```suggestion
with the NXP-provided BLE Link Layer Controller library. The library will need to
```

Reply:
I don't believe that is the proper use of a hyphen.